### PR TITLE
solution

### PR DESCRIPTION
**Pull request by OleksandrDzindziura**

### CODE REVIEW

**Review by sasha-morozov — APPROVED** [review]
Good job!

File: src/arrayMethodReduce.js [hunk 1]
```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
+    let startIndex = 0;
+
+    if (arguments.length < 2) {
```
Comment (VinnieJ-2k20) [inline src/arrayMethodReduce.js]: Let's use more obvious check here.
```suggestion
    if (initialValue === undefined) {
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Almost done, just fix one important thing.

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**